### ISSUE TITLE
Call list of certificates for a user via API

### ISSUE THREAD

**Issue by LaurentLMC** [mdjnelson/moodle-mod_customcert#419]
Hi, 

We are using the certificate module in moodle but our front-end site uses a different CMS and only makes calls to the moodle platform via the API. 

Is there  any way to send a request via the API to know if user X has any certificate available ?

Many thanks in advance for your help !

Lt

**Comment by mdjnelson** [comment]
This would require creating a web service. See https://docs.moodle.org/dev/Web_services. :) I don't have time to create that right now, sorry.

**Comment by karol-gro** [comment]
@mdjnelson I can see in our Moodle web service endpoints which (as I understand) already come from the customcert:
- mod_customcert_delete_issue 
- mod_customcert_get_element_html 
- mod_customcert_save_element 

So does it means that to get the list of issued certificates (for a user or total) it would be sufficient to add a method there?

**Comment by mdjnelson** [comment]
Yes those are the ones that I wrote in for the rearranging of elements page. If you create another one please create a PR I can review. :)
